### ISSUE TITLE
how to change base color

### ISSUE THREAD

**Issue by Vyruim** [zamsyt/obsidian-ion#3]
haallloooo, so i was wondering how to change the base color of the ion theme. This is by far my fav theme as a new Obsidian user and as someone learning CSS. Could you help me find out how to change the color?

**Comment by zamsyt** [comment]
Lovely to hear you like it! Would you like to change the text color, or use a different background color than black? Text color can be changed using the Style Settings plugin, but I haven't added support for different background colors yet.

**Comment by Vyruim** [comment]
> Lovely to hear you like it! Would you like to change the text color, or use a different background color than black? Text color can be changed using the Style Settings plugin, but I haven't added support for different background colors yet.

yep! was just looking for how to change text colors! ty so much for this! Hope yuu continue the great work!
